fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/TrafficManagementCenterUnit.java

```diff
@@ -44,7 +46,7 @@
 /**
  * This class represents a Traffic Management Center in the application simulator.
  */
-public class TrafficManagementCenterUnit extends AbstractSimulationUnit implements TrafficManagementCenterOperatingSystem {
+public class TrafficManagementCenterUnit extends ServerUnit implements TrafficManagementCenterOperatingSystem {
```

Comment:
I'm not sure if this change is helpful. 🤷‍♂️

Reply:
this seems to be very useful, as we now could configure that the TMC is deployed inside a server.